README.md

```diff
@@ -4,29 +4,73 @@
 
 Carthage is intended to be the simplest way to add frameworks to your Cocoa application.
 
-The basic [workflow](#adding-frameworks-to-an-application) looks something like this:
-
-1. Create a [Cartfile][] that lists the frameworks you’d like to use in your project.
-1. [Run Carthage](#adding-frameworks-to-an-application), which fetches and builds each framework you’ve listed.
-1. Drag the built `.framework` binaries into your application’s Xcode project.
-
 Carthage builds your dependencies and provides you with binary frameworks, but you retain full control over your project structure and setup. Carthage does not automatically modify your project files or your build settings.
 
-## Differences between Carthage and CocoaPods
+- [Quick Start](#quick-start)
+- [Installing Carthage](#installing-carthage)
+- [Adding frameworks to an application](#adding-frameworks-to-an-application)
+	- [Getting started](#getting-started)
+			- [If you're building for OS X](#if-youre-building-for-os-x)
+			- [If you're building for iOS, tvOS, or watchOS](#if-youre-building-for-ios-tvos-or-watchos)
+			- [For both platforms](#for-both-platforms)
+			- [(Optionally) Add build phase to warn about outdated dependencies](#optionally-add-build-phase-to-warn-about-outdated-dependencies)
+			- [Swift binary framework download compatibility](#swift-binary-framework-download-compatibility)
+	- [Running a project that uses Carthage](#running-a-project-that-uses-carthage)
+	- [Adding frameworks to unit tests or a framework](#adding-frameworks-to-unit-tests-or-a-framework)
+	- [Upgrading frameworks](#upgrading-frameworks)
+			- [Experimental Resolver](#experimental-resolver)
+	- [Nested dependencies](#nested-dependencies)
+	- [Using submodules for dependencies](#using-submodules-for-dependencies)
+	- [Automatically rebuilding dependencies](#automatically-rebuilding-dependencies)
+	- [Caching builds](#caching-builds)
+	- [Bash/Zsh/Fish completion](#bashzshfish-completion)
+- [Supporting Carthage for your framework](#supporting-carthage-for-your-framework)
+	- [Share your Xcode schemes](#share-your-xcode-schemes)
+	- [Resolve build failures](#resolve-build-failures)
+	- [Tag stable releases](#tag-stable-releases)
+	- [Archive prebuilt frameworks into one zip file](#archive-prebuilt-frameworks-into-one-zip-file)
+		- [Use travis-ci to upload your tagged prebuild frameworks](#use-travis-ci-to-upload-your-tagged-prebuild-frameworks)
```

Comment:
@blender Not your typo to begin with, but “prebuild” should be “prebuilt” both here and in the header.